frontend/app/(tabs)/home.tsx

```diff
@@ -29,24 +29,14 @@ const Index = () => {
   const SCROLL_PADDING = 20;
 
   const searchAnimation = useRef(new Animated.Value(0)).current;
+  const searchTimeout = useRef<NodeJS.Timeout>();
 
-  const filteredCards = useMemo(() => {
-    if (searchText.trim() === '') {
-      return cards;
-    } else {
-      return cards.filter(
-        (card) =>
-          card.title.toLowerCase().includes(searchText.toLowerCase()) ||
-          (card.content &&
-            card.content.toLowerCase().includes(searchText.toLowerCase())),
-      );
-    }
-  }, [cards, searchText]);
+  const filteredCards = useMemo(() => cards, [cards]);
```

Comment:
This doesn't seem to do anything, can you remove it and everything related to the filtered cards?